tests/data/placebo/test_datapipeline_reporting/datapipeline.DescribePipelines_1.json

```diff
@@ -0,0 +1,82 @@
+{
+    "status_code": 200, 
+    "data": {
+        "ResponseMetadata": {
+            "RetryAttempts": 0, 
+            "HTTPStatusCode": 200, 
+            "RequestId": "75644193-07e1-11e7-91e2-bfd5af170b83", 
+            "HTTPHeaders": {
+                "x-amzn-requestid": "75644193-07e1-11e7-91e2-bfd5af170b83", 
+                "date": "Mon, 13 Mar 2017 11:37:37 GMT", 
+                "content-length": "849", 
+                "content-type": "application/x-amz-json-1.1"
+            }
+        }, 
+        "pipelineDescriptionList": [
+            {
+                "fields": [
```

Comment:
I believe I've addressed this in d5c3aa985276a1aa4738bf59bf8dbb77501851ee, though the test is unchanged (still at `resource['tags']`) so I'm not sure whether I did this right and/or it's truly necessary. Is that commit what you had in mind?